crates/topos-p2p/src/error.rs

```diff
@@ -38,6 +38,9 @@ pub enum P2PError {
 
     #[error("Unable to execute shutdown on the p2p runtime: {0}")]
     ShutdownCommunication(mpsc::error::SendError<oneshot::Sender<()>>),
+
+    #[error("Unable to create gRPC client")]
+    UnableToCreateGrpcClient,
```

Comment:
Would be good to include the tonic error here. As it is it's not obvious what the problem was?